CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* Standalone servers now have a default socket timeout of 30 seconds, fixing
```

Comment:
Could you rebase/merge master, and move this entry to master section of this changelog?